src/mongocxx/test/private/write_concern.cpp

```diff
@@ -23,7 +23,6 @@ using namespace mongocxx;
 
 TEST_CASE("creation of write_concern passes universal parameters to c-driver's methods",
           "[write_concern][base][c-driver]") {
```

Comment:
This is because `KeepEmptyLinesAtTheStartOfBlocks: false`. Since this is the start of a block, the blank line is removed. The same setting is used in the server.